docs/pages/docs/apis/access-control.mdx

```diff
@@ -268,55 +195,32 @@ export default config({
     ListKey: list({
       fields: {
         fieldName: text({
-          // Imperative access control definition
-          access: args => true,
+          access: {
+            read: ({ session, context, listKey, fieldKey, operation, item }) => true,
+            create: ({ session, context, listKey, fieldKey, operation, originalInput }) => true,
```

Comment:
Wait, `create` gets `originalInput` with no item? Consistency matching bikeshed conversation?

Reply:
That's correct. When you `create` an object, it doesn't already exist, so `item` has no meaning for that operation.